docs/demos/podman-ai-lab-to-rhoai/podman-ai-lab-to-rhoai.md

```diff
@@ -0,0 +1,447 @@
+
+1. Go to the AI Lab extension and select Catalog under Models.
+![Catalog](img/AI_lab_catalog.png)
+
+2. If you haven't already, download the _TheBloke/Mistral-7B-Instruct-v0.2-GGUF_ model.
+![Download Model](img/mistral_download.png) 
++ The model is around 4GB so it might take some time. 
+![Download Progress](img/mistral_download_progress.png)
+
+3. Podman AI Lab allows you to get started quickly with downloaded models through Services, Playgrounds, and the Recipes Catalog. 
+      1. **Services** -
+      The Services section allows you to create a model service endpoint for models you've downloaded. Client code is provided (cURL by default) in multiple formats to get you started quickly with sending in requests to the model service endpoint. 
+      ![Podman AI Lab Services](img/podman_AI_services.png)
+
+      1. **Playgrounds** -
+      The Playgrounds area allows you to define system prompts and experiment with different settings like temperature, max tokens, and top-p.
+      ![Podman AI Lab Playgrounds](img/podman_AI_playground.png)
+
+      1. **Recipes Catalog** -
+      The Recipes Catalog contains demo applications for Natural Language Processing (NLP), Computer Vision, and Audio. We'll be using the ChatBot recipe demo code in this example.
+
+5. **Create the Chatbot** -
+Make sure to select _TheBloke/Mistral-7B-Instruct-v0.2-GGUF_ as your model and then click **_Start AI App_** button.
+![Start Chatbot](img/start_chatbot_recipe.png)
+
+6. After the chatbot has started open it up to test it out.
+![Chatbot UI](img/chatbot_recipe_UI.png)
+
+7. At the bottom of the AI App Details section you'll see a **_Open in VSCode_** button. Clicking on that will open all of the code that is running your chatbot. Later we'll modify that code to connect langchain, TheBloke/Mistral-7B-Instruct-v0.2-GGUF model, and the Elasticsearch Vector Database. 
+![Podman AI Chatbot Recipe](img/podman_AI_chatbot.png)
+
+## Deploying OpenShift AI
+
+1. Clone [podman-ai-lab-to-rhoai](https://github.com/redhat-ai-services/podman-ai-lab-to-rhoai){:target="_blank"}
+
+2. Login to your OpenShift cluster in a terminal with the API token. You can get your API token from the OpenShift web console.
+    ![OpenShift API Login Command](img/ocp_api_login.png) 
+    ```
+    oc login --token=<YOUR_OPENSHIFT_API_TOKEN> --server=https://<YOUR_OPENSHIFT_API_URL>:6443
+    ```
+
+3. We'll first deploy the OpenShift AI operator. 
+    ```
+    oc apply -k ./components/openshift-ai/operator/overlays/fast
+    ```
+
+4. Now we'll create a Data Science Cluster. Make sure the operator is fully deployed before creating the Data Science Cluster.
+    ```
+    watch oc get pods -n redhat-ods-operator
+    ```
+
+    ![OpenShift AI Operator Status](img/oc_rhoai_operator_status.png)
+
+    Once the pod has a Running status and is ready you can run the below command.
+    + ```
+    oc apply -k ./components/openshift-ai/instance/overlays/fast
+    ```
+
+## Deploy Elasticsearch Vector DB
```

Comment:
Same as before, I would set this in its own repo/doc or mark it optional.

Reply:
Made the section optional, included the installation instructions in the operator documentation, and made the automated installation collapsible.